### ISSUE TITLE
Weird Bullet patterns, possibly unintended?

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#563]
Not sure if that's intended or no but it seems like Little Horn is causing alt path's It Lives to use patterns that weren't normally intended

which wouldn't be so bad if the bullets didn't go offscreen and came back in a way really hard to predict in like 2 seconds

might be a skill issue, but i really doubt its intended so sharing how it looked just in case

https://www.twitch.tv/cool__g/clip/DarkDrabAniseAMPEnergy-C5NbWNYVhJ7pWqKo

**Comment by Neonomi** [comment]
this pattern is so cursed

**Comment by Lonrite** [comment]
> this pattern is so cursed

yeah, i was kinda shocked but at least it was early in the streak

ya bi dazhe skasal chto polniy pizdec a ne attacka)

**Comment by AyyItsClarkster** [comment]
Those are the normal bullet patterns of Mausoleum Mom's Heart. It's supposed to be considerably harder than the regular Mom's Heart/It Lives.

**Comment by Rchardon** [comment]
@AyyItsClarkster no, the way the bullet are moving in the end definitely isn't a normal pattern of alt Mom's heart

**Comment by Lonrite** [comment]
> Those are the normal bullet patterns of Mausoleum Mom's Heart. It's supposed to be considerably harder than the regular Mom's Heart/It Lives.

yeah, I wish I could agree but sadly something's up with those

little horn probably fucks it up, and ye its not normal at all

**Comment by Neonomi** [comment]
i just noticed hush patterns is also broken, even player tears is glitchy with continuum

**Comment by Lonrite** [comment]
> i just noticed hush patterns is also broken, even player tears is glitchy with continuum

oh damn, that's bad news but thanks for notifying at least, I had no clue about that I think

if you mean little horn that is
